README.md

```diff
@@ -226,6 +226,26 @@ query: {
 }
 ```
 
+### $sqs
+[simple_query_string](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html). A query that uses the SimpleQueryParser to parse its context. Optional `$default_operator` which is set to `or` by default but can be set to `and` if required.
+
+```js
+query: {
+  $sqs: {
+    $fields: [
+      'title^5',
+      'description'
+    ],
+    $query: '+like +javascript',
+    $default_operator: 'and'
```

Comment:
It would be great if we could change `$default_operator` to simply `$operator`. I try to keep the parameters as short as possible, as the queries tend to grow quite long quite quickly.